angular.json

```diff
@@ -114,9 +126,156 @@
                 }
               ],
               "fileReplacements": [
+                {
+                  "replace": "projects/storefrontapp/src/environments/occ/occ.environment.ts",
+                  "with": "projects/storefrontapp/src/environments/occ/occ.environment.ccv2.ts"
+                },
+                {
+                  "replace": "projects/storefrontapp/src/environments/environment.ts",
+                  "with": "projects/storefrontapp/src/environments/environment.prod.ts"
+                }
+              ],
+              "optimization": true,
+              "outputHashing": "all",
+              "sourceMap": false,
+              "extractCss": true,
+              "namedChunks": false,
+              "aot": true,
+              "extractLicenses": true,
+              "vendorChunk": false,
+              "buildOptimizer": true,
+              "tsConfig": "projects/storefrontapp/tsconfig.app.prod.json"
+            },
+            "cds.development": {
+              "budgets": [
+                {
+                  "type": "anyComponentStyle",
+                  "maximumWarning": "6kb"
+                }
+              ],
+              "fileReplacements": [
+                {
+                  "replace": "projects/storefrontapp/src/environments/cds/cds.environment.ts",
+                  "with": "projects/storefrontapp/src/environments/cds/cds.environment.dev.ts"
+                },
+                {
+                  "replace": "projects/storefrontapp/src/environments/occ/occ.environment.ts",
+                  "with": "projects/storefrontapp/src/environments/occ/occ.environment.1905.ts"
+                }
+              ]
+            },
+            "cds.production": {
+              "budgets": [
+                {
+                  "type": "anyComponentStyle",
+                  "maximumWarning": "6kb"
+                }
+              ],
+              "fileReplacements": [
+                {
+                  "replace": "projects/storefrontapp/src/environments/cds/cds.environment.ts",
+                  "with": "projects/storefrontapp/src/environments/cds/cds.environment.dev.ts"
+                },
+                {
+                  "replace": "projects/storefrontapp/src/environments/environment.ts",
+                  "with": "projects/storefrontapp/src/environments/environment.prod.ts"
```

Comment:
And similarly in production we should use occ-1905 environemnts

Reply:
Good catch.